applications/serial_lte_modem/src/slm_at_host.c

```diff
@@ -658,6 +658,14 @@ AT_MONITOR(at_notify, ANY, notification_handler);
 static void notification_handler(const char *notification)
 {
 	if (get_slm_mode() == SLM_AT_COMMAND_MODE) {
+
+#if defined(CONFIG_SLM_PPP)
+		if (!slm_fwd_cgev_notifs
+		 && !strncmp(notification, "+CGEV: ", strlen("+CGEV: "))) {
+			/* CGEV notifications are silenced. Do not forward them. */
+			return;
+		}
+#endif
```

Comment:
```suggestion
		if (IS_ENABLED(CONFIG_SLM_PPP) && !slm_fwd_cgev_notifs
		    && !strncmp(notification, "+CGEV: ", strlen("+CGEV: "))) {
			/* CGEV notifications are silenced. Do not forward them. */
			return;
		}
```

I would prefer `IS_ENABLED()` over `#if defined` on a code that can actually compile when the feature is disabled. Compiler will then still do the syntax checking on the disabled part, even when its going to be finally optimized away.

Reply:
True. Though in that case the variable is defined in `slm_ppp.h` that is only included `#if defined(CONFIG_SLM_PPP)`.

Reply:
You can move the variable as well. As it is unused, compiler should drop it.

Reply:
Yeah, though I have a comment on its definition that pertains to PPP...